cookbook/navigation/navigation-basics.md

```diff
@@ -0,0 +1,160 @@
+---
+layout: page
+title: "Navigate to a new screen and back"
+permalink: /cookbook/navigation/navigation-basics/
+---
+
+Most apps contain several screens for displaying different types of information. 
+For example, we might have a screen that shows products. Our users could then
+tap on a product to get more information about it on a new screen.
```

Comment:
Added "In Android terms, this would be a new Activity. In iOS terms, a new ViewController. In Flutter, a screen is just a another Widget!"

I'm not so great with iOS, so please let me know if that language is right.